src/internal/m365/service/groups/export.go

```diff
@@ -33,19 +39,48 @@ func ProduceExportCollections(
 			fp      = restoreColl.FullPath()
 			cat     = fp.Category()
 			folders = []string{cat.String()}
+			coll    export.Collectioner
 		)
 
 		switch cat {
 		case path.ChannelMessagesCategory:
 			folders = append(folders, fp.Folders()...)
-		}
 
-		coll := groups.NewExportCollection(
-			path.Builder{}.Append(folders...).String(),
-			[]data.RestoreCollection{restoreColl},
-			backupVersion)
+			coll = groups.NewExportCollection(
+				path.Builder{}.Append(folders...).String(),
+				[]data.RestoreCollection{restoreColl},
+				backupVersion)
+		case path.FilesCategory:
+			drivePath, err := path.ToDrivePath(restoreColl.FullPath())
+			if err != nil {
+				return nil, clues.Wrap(err, "transforming path to drive path").WithClues(ctx)
+			}
+
+			driveName, ok := backupDriveIDNames.NameOf(drivePath.DriveID)
+			if !ok {
+				// This should not happen, but just in case
+				logger.Ctx(ctx).With("drive_id", drivePath.DriveID).Info("drive name not found, using drive id")
+				driveName = drivePath.DriveID
+			}
+
+			baseDir := path.Builder{}.
+				Append("Libraries").
+				Append(driveName).
+				Append(drivePath.Folders...)
 
-		ec = append(ec, coll)
+			coll = drive.NewExportCollection(
+				baseDir.String(),
+				[]data.RestoreCollection{restoreColl},
+				backupVersion)
+		default:
+			el.AddRecoverable(
+				ctx,
+				clues.New("unsupported category for export").With("category", cat))
+		}
```

Comment:
```suggestion
				clues.New("unsupported category for export").With("category", cat))
				
				continue
		}
```

Reply:
LOL, not sure why I added a `nil` check instead of this. Good catch.